WebViewControl/ResourcesManager.cs

```diff
@@ -19,8 +19,28 @@ public static class ResourcesManager {
             return InternalTryGetResource(assembly, defaultNamespace, resourcePath, failOnMissingResource);
         }
 
+        private static string ComputeResourceName(string defaultNamespace, IEnumerable<string> resourcePath) {
+            var resourcePathSize = resourcePath.Count();
```

Comment:
var resourceParts = (new[] { defaultNamespace}).Concat(resourcePath);
for (int i = 0; i < resourceParts.Length - 1; i++) {
    resourceParts[i] = resourceParts[i].Replace('-', '_');
}

return string.Join(".", resourceParts);